pints/_abc/_abc_rejection.py

```diff
@@ -0,0 +1,89 @@
+#
+# ABC Rejection method
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+import numpy as np
+
+import pints
+
+
+class RejectionABC(pints.ABCSampler):
+    """
+    Rejection ABC sampler.
+
+    See, for example, [1]_. In each iteration of the algorithm, the following
```

Comment:
I don't like the "for example" here. Is this ref the _exact_ version we implement or not?